File: MORM.Java/src/MORM/Java/Models/Venda/Ncm.java
package MORM.Java.Models.Venda;

import java.util.Date;
import MORM.Java.Classes.*;
import MORM.Java.Classes.Mapping.*;

@TabelaAttribute(Nome = "NCM")
public class Ncm extends CollectionItem
{
    @CampoAttribute(Campo = "CD_NCM", Tipo = CampoTipo.Key)
    public String Cd_Ncm;
    @CampoAttribute(Campo = "U_VERSION", Tipo = CampoTipo.Nul)
    public String U_Version;
    @CampoAttribute(Campo = "CD_OPERADOR", Tipo = CampoTipo.Req)
    public Integer Cd_Operador;
    @CampoAttribute(Campo = "DT_CADASTRO", Tipo = CampoTipo.Req)
    public Date Dt_Cadastro;
    @CampoAttribute(Campo = "DS_NCM", Tipo = CampoTipo.Req)
    public String Ds_Ncm;
}

File: MORM.CSharp/MORM.Dominio/Models/Venda/Ncm.cs
using MORM.Util.Classes;
using System;

namespace MORM.CSharp.Models
{
    [Tabela("NCM")]
    public class Ncm : CollectionItem
    {
        [Campo("CD_NCM", CampoTipo.Key)]
        public string Cd_Ncm { get; set; }
        [Campo("U_VERSION", CampoTipo.Nul)]
        public string U_Version { get; set; }
        [Campo("CD_OPERADOR", CampoTipo.Req)]
        public int Cd_Operador { get; set; }
        [Campo("DT_CADASTRO", CampoTipo.Req)]
        public DateTime Dt_Cadastro { get; set; }
        [Campo("DS_NCM", CampoTipo.Req)]
        public string Ds_Ncm { get; set; }
    }
}